test/extended/builds/start.go

```diff
@@ -51,6 +51,25 @@ var _ = g.Describe("[builds][Slow] starting a build using CLI", func() {
 		})
 	})
 
+	g.Describe("setting OwnerReferences", func() {
+		g.It("should set parent BuildConfig as OwnerReference", func() {
+			g.By("starting the build from buildconfig")
+			br, err := exutil.StartBuildAndWait(oc, "sample-build")
+			o.Expect(err).NotTo(o.HaveOccurred())
+			br.AssertSuccess()
+			if len(br.Build.ObjectMeta.OwnerReferences) == 0 {
+				o.Expect(fmt.Errorf("Created build has no OwnerReferences")).NotTo(o.HaveOccurred())
+			}
```

Comment:
you should make sure the reference references the right thing (proper type, name of the buildconfig matches what you'd expect)